ember-client/src/main/scala/org/http4s/ember/client/internal/ClientHelpers.scala

```diff
@@ -223,6 +223,7 @@ private[client] object ClientHelpers {
         // case Left(EmberException.EmptyStream()) => true // Next version can be accessed by users
         case Left(org.http4s.ember.core.EmptyStreamError()) =>
```

Comment:
i'm guessing this match clause isn't necessary anymore but deferring to @ChristopherDavenport on that one

Reply:
Yeah I wasn't sure as to whether we should keep that in there or not.  It's not currently getting matched on.

Reply:
It looks to me like it can only be raised where it has been adapted.  I think it can go.